pkg/sql/colexec/colbuilder/execplan.go

```diff
@@ -539,9 +540,9 @@ func (r opResult) createAndWrapRowSource(
 	if args.ProcessorConstructor == nil {
 		return errors.New("processorConstructor is nil")
 	}
-	log.VEventf(ctx, 1, "planning a row-execution processor in the vectorized flow because %v", causeToWrap)
+	log.VEventf(ctx, 1, "planning a row-execution processor in the vectorized flow because %v", redact.Safe(causeToWrap))
```

Comment:
Make causeToWrap a redact.RedactableString and prepare redactability at the source.